CHANGES.rst

```diff
@@ -1,9 +1,13 @@
 Changelog
 =========
 
-3.7 (unreleased)
+4.0 (unreleased)
 ----------------
 
+- Add support for Python 3.11.
```

Comment:
`CHANGES.rst` says `Add support for Python 3.11` but in the diff I have not seen 3.11 additions (may have been already there).

Reply:
Python 3.11 support was already there, I removed the entry.